### PR TITLE
Phase

### LINKED ISSUE #6: Phase

### PR DESCRIPTION
**Pull request by valcobe**

### CODE REVIEW

**Comment by rogargon** [comment]
There are some code style errors detected by `ng lint` that need fixing before PR acceptance:

```
Linting "academicrecruit-gei"...

/home/runner/work/AcademicRecruit-GEI/AcademicRecruit-GEI/src/app/qualification/qualification-edit/qualification-edit.component.ts:23:43
Lint errors found in the listed files.
ERROR: 23:43  semicolon  Missing semicolon

/home/runner/work/AcademicRecruit-GEI/AcademicRecruit-GEI/src/app/qualification/qualification-list/qualification-list.component.ts:20:5
ERROR: 20:5   align      parameters are not aligned
```

**Comment by rogargon** [comment]
Phases are not created (404 error). Please, check the implemented features are actually working as expected

File: src/app/phase/phase-create/phase-create.component.html [hunk 1]
```diff
@@ -1 +1,26 @@
...
+<form (ngSubmit)="onSubmit()" #form="ngForm">
+  <fieldset>
+    <div class="form-group">
+      <div>
+        <p>phaseNumber</p>
+        <input id="phasenumber" name="phasenumber" class="form-control" required [(ngModel)]="phase.phasenumber"/>
+      </div>
+      <div>
+        <p>Name</p>
+        <input id="name" name="name" class="form-control" [(ngModel)]="phase.phasenumber"/>
```
Comment (bergacat1) [inline src/app/phase/phase-create/phase-create.component.html]: This field should be related to `phase.name`.

File: src/app/phase/phase-create/phase-create.component.html [hunk 2]
```diff
@@ -1 +1,26 @@
-<button class="btn btn-primary cmbutton" routerLink="/phase">Create Phase</button>
+<form (ngSubmit)="onSubmit()" #form="ngForm">
+  <fieldset>
+    <div class="form-group">
+      <div>
+        <p>phaseNumber</p>
+        <input id="phasenumber" name="phasenumber" class="form-control" required [(ngModel)]="phase.phasenumber"/>
```
Comment (bergacat1) [inline src/app/phase/phase-create/phase-create.component.html]: This field is called `phaseNumber` in the API and, thus, it should be called the same at Phase class of the client side.

**Review by bergacat1 — DISMISSED** [review]
Phase's `phaseNumber` and `name` are not stored. Moreover, you could indicate the format of the dates (ideally you should place a date picker).

**Review by rogargon — APPROVED** [review]

**Pull request merged**